packages/client/components/NewGitLabIssueInput.tsx

```diff
@@ -127,16 +126,13 @@ const NewGitLabIssueInput = (props: Props) => {
                   }
                   path
                 }
-                query {
-                  # use alias otherwise projects args must be the same as GitLabScopingSearchResults query
-                  allProjects: projects(
-                    membership: true
-                    first: 100
-                    sort: "latest_activity_desc"
-                  ) {
+                # use alias to tell relay that this query shouldn't be cached with GitLabScopingSearchResults query
+                newIssueQuery: query {
```

Comment:
I ran into the same problem where adding the `selectedProjectsIds` filter to `GitLabScopingSearchResults` resulted in the new issue input query getting filtered when we don't want it to. Changing the `projects` args didn't work, but aliasing the query did.